### PR TITLE
Simple cookie hint

### PR DESCRIPTION
**Pull request by dmorley**
https://github.com/diasporg/Poduptime/issues/149

Very basic to try and help users remember the pod they picked (last in the simple case)

### CODE REVIEW

File: go.php [hunk 1]
```diff
@@ -48,7 +48,7 @@
     if (!(new CrawlerDetect())->isCrawler()) {
         R::store($c);
     }
-
+    setcookie("domain", $domain, 2147483647);
```
Comment (noplanman) [inline go.php]: Erm, expiry should be in relation to now, no?
As in `Carbon::now()->addCentury()->timestamp;`

File: show.php [hunk 2]
```diff
@@ -47,8 +47,11 @@
         $pod_name       = htmlentities($pod['name'], ENT_QUOTES);
         $humanmonitored = Carbon::now()->subDays($pod['daysmonitored'])->diffForHumans(null, true);
         $tip            = "This {$pod['softwarename']} pod's uptime is {$pod['uptime_alltime']}% over {$humanmonitored}.";
-        echo '<tr><td><div title="' . $tip . '" data-toggle="tooltip" data-placement="bottom"><a class="text-success url" target="_self" href="/go.php?domain=' . $pod['domain'] . '">' . $pod['domain'] . '</a></div></td>';
-
+        if ($_COOKIE["domain"] === $pod['domain']) {
```
Comment (noplanman) [inline show.php]: If the cookie hasn't been set yet, this throws an error, so fix it with:
```php
...
if ($_COOKIE['domain'] ?? null === $pod['domain']) {
...
```

:exclamation: Same applies to `showfull.php`

**Comment by noplanman** [comment]
Might be good to have a popup asking about saving the cookie?

Hmm, or maybe because this cookie is actually a true helper, just let it do it's job :wink:

**Comment by dmorley** [comment]
Let's not ask, but we could put one of them cookie things saying we use them? the sorter also sets several cookies now to remember your sort

**Comment by noplanman** [comment]
> we could put one of them cookie things saying we use them?

Yep :+1:

**Pull request merged**